lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -283,14 +283,6 @@ def bootstrap(proto=nil)
           warn_chef_config_secret_key
         end
 
-        bootstrap_architecture = Chef::Config[:knife][:bootstrap_architecture]
-        if bootstrap_architecture && ![:x86_64, :i386].include?(bootstrap_architecture.to_sym)
-          raise "Valid values for the knife config :bootstrap_architecture are i386 or x86_64. Supplied value is #{bootstrap_architecture}"
-        end
-        if Chef::Config[:knife][:architecture]
-          raise "Do not set :architecture in your knife config, use :bootstrap_architecture."
-        end
-
         validate_name_args!
```

Comment:
Not sure why the user should give architecture as an input when every server will have a fixed architecture and we can find it ourselves. Taking input from the user increases the risk of passing wrong value too.
Hence removed this.